### ISSUE TITLE
Duplicate entries in pool-list

### ISSUE THREAD

**Issue by fsouza** [tsuru/tsuru#1415]
For some reason, the public pool appears twice in the list:

```
% tsuru pool-list
+----------------+-------------+--------+
| Pool           | Teams       | Kind   |
+----------------+-------------+--------+
| mypool         |             | public |
| mypool         |             | public |
| yourpool-debug | secret-team |        |
| yourpool-temp  | secreat-team|        |
+----------------+-------------+--------+
```

**Comment by fsouza** [comment]
This is fixed on master (here is the broken code: https://github.com/tsuru/tsuru/blob/1.0.0-rc6/api/pool.go#L48-L56).

Can we include the fix in tsuru 1.0? :D

**Comment by andrewsmedina** [comment]
It will be included in tsuru 1.0

**Comment by andrewsmedina** [comment]
fixed in 1.0.0rc7
